components/ai_chat/resources/page/state/conversation_context.tsx

```diff
@@ -282,6 +282,12 @@ export function ConversationContextProvider(
     )
     listenerIds.push(id)
 
+    id = callbackRouter.onConversationDeleted.addListener(() => {
+      // TODO(petemill): Show deleted UI
+      console.debug('DELETED')
```

Comment:
@petemill do we need to do something more here?

Reply:
I was leaving it open for us to display a note, something like "this conversation has been deleted", or switch to just switch to a new conversation